src/algebraic_geometry/Gamma_Spec_adjunction.lean

```diff
@@ -82,7 +82,7 @@ abbreviation opens_map_basic_open (r : Γ' X) :=
   (opens.map X.to_Γ_Spec_base).obj (basic_open r)
 
 /-- The preimage is a basic open in `X` defined by the same element `r`. -/
-lemma to_Γ_Spec_opens_map_obj_basic_open_eq (r : Γ' X) :
+lemma to_Γ_Spec_opens_map_basic_open_eq (r : Γ' X) :
```

Comment:
I think the naming convention would suggest `to_Γ_Spec_opens_map_eq_basic_open`.

Reply:
It's stating `opens.map` of a basic open is still a basic open. Maybe the full name should be `to_Γ_Spec_map_eq_basic_open_eq_basic_open` but I now adopt `to_Γ_Spec_map_eq_basic_open_eq` which is consistent with `to_Γ_Spec_preim_basic_open_eq` above.